### ISSUE TITLE
Have Scrabble robot use Webster's Dictionary or difficulty levels

### ISSUE THREAD

**Issue by unarmedpawn** [cdot/Xanado#106]
**Is your feature request related to a problem? Please describe.**
Not really the robot just uses words that aren't showing up in my online dictionaries and my challenges are never honored

**Describe the solution you'd like**
For the robot to use more legitimate words or have a difficulty setting where we can dumb it down for me a bit

**Describe alternatives you've considered**
Cheating alongside the robot as my made up words are also never challenged. This kind of ruins the whole game. 

**Additional context**
Here are the questionable words used this game: oba, yabas, ayries, ronz. It is difficult to play with integrity against these words.

**Comment by cdot** [comment]
Thank you for your report.

Xanado has a range of dictionaries, most of which have been sourced online (except for British_English, which I created). It sounds like you have been playing with one of the official Scrabble dictionaries (CSW_2019 or SOWPODS) which have a lot of obscure (though perfectly valid) words in them. Xanado doesn't have any measure of the difficulty of a word, it simply plays what it finds in the dictionary.

I don't speak American, so I would never use Webster's myself. But if you can find a word list for it then it's easy enough to generate your own dictionary; full instructions are in the README.

Later: I see one [here](https://webster-dictionary.net/Index-a) but it contains a lot of nouns (capitalised words) and phrases that would have to be pruned before it could be used for Scrabble.

I have tested the challenge mechanism and it is working fine; as long as you select that option when setting up the game, the robot will challenge words not found in the dictionary.
